assets/component-facets.css

```diff
@@ -331,18 +336,112 @@
 .facets-layout-grid .facets__label {
   display: flex;
   flex-direction: column;
-  gap: 0.8rem;
   padding: 0;
   height: 100%;
+  min-width: 0;
```

Comment:
`min-width: 0` - I can't figure out what this was doing. The UI went through so many iterations, maybe it was something that was previously needed, but I can't figure out why it would be needed now. I can remove it.

`height: 100%;` - this is needed because some text may wrap to multiple lines. We want all labels in a row to match height. Without this, you can see the outline does not go to the bottom where it should.

With height: 100%: 
<img width="412" alt="image" src="https://github.com/Shopify/dawn/assets/1019769/3ed31d79-3200-478d-88cb-9c28ef2fbe33">


Without it: 
<img width="453" alt="image" src="https://github.com/Shopify/dawn/assets/1019769/b6824bff-772a-4942-96a9-6b8bb1bd6d94">